### PR TITLE
CXC-366: update catalog with new structure

### PR DESCRIPTION
**Pull request by Vanessi-K**
Please check the catalog and all associated functionality. Please verify if the new section in the Readme can be easily understood.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,3 +65,26 @@ always define it in the `_variables.scss` file.
...
+
+## Updating the catalog
+The catalog-structure is defined in `server/assets/comixplain-catalog.js`. 
+This catalog follows the same structure as in the [Comixplain repository](https://github.com/fhstp/comixplain/blob/main/docs/assets.json).
+The structure in the `comixplain-catalog.js` is used to generate the catalog on the website. 
+The naming of categories, sub-categories and filters all originate from this catalog, 
+therefore it is important that all keys carry readable names.
+
+### Updating the catalog with a new version from the Comixplain repository
+This paragraph describes how the existing catalog structure from the Comixplain repository most be adjusted to
```
Comment (hannahwagner1402) [inline README.md:77]: maybe instead of "this paragraph" you can say "these steps will describe..."

File: README.md [hunk 2]
```diff
@@ -65,3 +65,26 @@ always define it in the `_variables.scss` file.
...
+This catalog follows the same structure as in the [Comixplain repository](https://github.com/fhstp/comixplain/blob/main/docs/assets.json).
+The structure in the `comixplain-catalog.js` is used to generate the catalog on the website. 
+The naming of categories, sub-categories and filters all originate from this catalog, 
+therefore it is important that all keys carry readable names.
+
+### Updating the catalog with a new version from the Comixplain repository
+This paragraph describes how the existing catalog structure from the Comixplain repository most be adjusted to 
+properly work for comixcraft.
+
+1. Add the JSON-structure from the Comixplain repository (`/docs/assets.json`) to be
```
Comment (hannahwagner1402) [inline README.md:80]: I dont fully understand the first step, do you mean copy the json structure from the repository to then import it into comixplain-catalog.js?

Comment (Vanessi-K) [reply README.md:80]: Yes

Comment (hannahwagner1402) [reply README.md]: the clairification in the readme is great, thanks!

**Pull request merged**